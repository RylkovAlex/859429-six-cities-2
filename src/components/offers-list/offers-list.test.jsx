import React from 'react';
import renderer from 'react-test-renderer';
import OfferList from './offers-list';

jest.mock(`../offer-card/offer-card.jsx`, () => jest.fn().mockReturnValue(null));

it(`OfferList correctly renders after relaunch`, () => {
  const props = {
    children: <div/>,
    className: ``,
  };

  const tree = renderer
    .create(<OfferList {...props}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
