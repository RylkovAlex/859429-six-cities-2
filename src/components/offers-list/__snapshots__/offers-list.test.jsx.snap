// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferList correctly renders after relaunch 1`] = `
<div
  className=""
>
  <div />
</div>
`;
